app-text/htmlrecode/htmlrecode-1.3.1-r1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Recodes HTML file using a new character set"
+HOMEPAGE="https://bisqwit.iki.fi/source/htmlrecode.html"
+SRC_URI="https://bisqwit.iki.fi/src/arch/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~ppc ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+
+PATCHES=( "${FILESDIR}/${P}-ar.patch" )
+HTML_DOCS=( README.html )
+
+src_prepare() {
+	touch .depend argh/.depend
+	default
+}
+
+src_configure() { :; }
```

Comment:
why?

Reply:
@Zlogene 
I guess, you're asking why i'm skipping ```src_configure``` at all? Because this already exists in the original ebuild too, i've just put it into one line...

Actually, I've tried to remove that part from the ebuild, but it's really not needed since the ```configure``` script is just a bash script which test's the compiler. It doesn't fail, but it's not needed at all.